temporal-sdk/src/main/java/io/temporal/client/WorkflowClient.java

```diff
@@ -572,6 +572,289 @@ static <A1, A2, A3, A4, A5, A6, R> WorkflowExecution start(
     return WorkflowClientInternalImpl.start(workflow, arg1, arg2, arg3, arg4, arg5, arg6);
   }
 
+  /**
+   * Executes a zero argument workflow update with a void return type
+   *
+   * @param updateMethod method reference annotated with @UpdateMethod of a proxy created through
+   *     {@link WorkflowClient#newWorkflowStub(Class, WorkflowOptions)}.
+   * @param options update options
+   * @return WorkflowUpdateHandle that can be used to get the result of the update
+   */
+  static WorkflowUpdateHandle<Void> executeUpdate(
```

Comment:
IMO this should be called `startUpdate` even though I know Java is not that consistent here. I think it's clearer to understand and allows us to offer a real `executeUpdate` later if we wanted. I'd also support `update`.

Reply:
Yep seems there is consensus that we should call this `startUpdate` so I renamed it.